dev/integration_tests/ios_add2app_life_cycle/README.md

```diff
@@ -8,21 +8,21 @@ The following functionality is currently implemented:
 
 1. A regular iOS view controller (UIViewController), similar to the default
    `flutter create` template (NativeViewController.m).
-1. A FlutterViewController subclass that takes over full screen. Demos showing
+2. A FlutterViewController subclass that takes over the full screen. Demos showing
```

Comment:
Please keep the numbers as `1.`'s, so they can be re-ordered, deleted, or inserted.

Reply:
Ah okay I thought it was a copy\paste mistake